upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -668,7 +809,9 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 			"pre-k8s-1.6": "2.4.2-kops.1",
 			"k8s-1.6":     "2.6.9-kops.1",
 			"k8s-1.7":     "2.6.9-kops.1",
+			"k8s-1.12":    "2.6.9-kops.1",
```

Comment:
Not sure whether there'd still be an explicit need to keep this / run on 1.12+ with calico v2.6.9? Maybe drop in favour of the new v3.4 manifest which applies with `KubernetesVersion: ">=1.12.0"`, so Calico `MajorVersion` is effectively defaulted to v3 for k8s v1.12+.

Reply:
not knowing the history behind having two versions available for 1.7, I wasn't sure if this was a python 2/3 sort of thing where it would make sense to keep both available for 1.12.

I'm happy to collapse onto the v3 version though